modules/oc-by-example-content.adoc

```diff
@@ -1651,6 +1722,14 @@ Mirror images from one repository to another
   # Note the above command is equivalent to
   oc image mirror myregistry.com/myimage:latest=myregistry.com/other:test \
   --filter-by-os=.*
+  
+  # Copy specific os/arch manifest of a multi-architecture image
+  # Run 'oc image info myregistry.com/myimage:latest' to see available os/arch for multi-arch images
+  # Note that the target registry may reject a manifest list if the platform specific images do not all
+  # exist. You must use a registry with sparse registry support enabled.
+  oc image mirror myregistry.com/myimage:latest=myregistry.com/other:test \
+  --filter-by-os=os/arch \
+  --keep-manifest-list=true
```

Comment:
the command is incorrect , we can't specify the os and keep-manifest-list at the same time:
oc image mirror  docker.io/library/busybox:latest=localhost:5000/other:test --filter-by-os=linux/386  --keep-manifest-list=true --insecure
localhost:5000/
  other
    blobs:
      docker.io/library/busybox sha256:44ddfaac8324c393489db23dbbb2a2f4ae18d36a86f1234a4c6bb16e459b5ca0 1.421KiB
      docker.io/library/busybox sha256:494467eb786caa44e77496badb070b5cf4350de34c72e5b1274bcf628603947e 2.163MiB
    manifests:
      sha256:4c6415d8307ac0555e20a047b83678d99063c0e9e89355541e8676d1d98f66a7
      sha256:3fbc632167424a6d997e74f52b878d7cc478225cffac6bc977eedfe51c7f4e79 -> test
  stats: shared=0 unique=2 size=2.165MiB ratio=1.00

phase 0:
  localhost:5000 other blobs=2 mounts=0 manifests=2 shared=0

info: Planning completed in 4s
uploading: localhost:5000/other sha256:494467eb786caa44e77496badb070b5cf4350de34c72e5b1274bcf628603947e 2.163MiB
sha256:4c6415d8307ac0555e20a047b83678d99063c0e9e89355541e8676d1d98f66a7 localhost:5000/other
error: unable to push manifest to localhost:5000/other:test: errors:
manifest blob unknown: blob unknown to registry
manifest blob unknown: blob unknown to registry
manifest blob unknown: blob unknown to registry
manifest blob unknown: blob unknown to registry
manifest blob unknown: blob unknown to registry
manifest blob unknown: blob unknown to registry
manifest blob unknown: blob unknown to registry
manifest blob unknown: blob unknown to registry
manifest blob unknown: blob unknown to registry

info: Mirroring completed in 9.43s (240.6kB/s)
error: one or more errors occurred while uploading images